<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>769</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Product Review Summarization System</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 228, 247);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(beijing.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>591</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Times New Roman&quot;;
background-color: rgba(204, 204, 204, 0);</string>
    </property>
    <property name="text">
     <string>Product Review Summarization System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>NLP  Lab ,   Soochow University</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>811</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_load">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Please load the data........</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>load data :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.985, y1:0.558, x2:1, y2:0, stop:0.189055 rgba(255, 255, 255, 255), stop:0.721393 rgba(200, 226, 246, 255));</string>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Select Product :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.985, y1:0.558, x2:1, y2:0, stop:0.189055 rgba(255, 255, 255, 255), stop:0.721393 rgba(200, 226, 246, 255));</string>
    </property>
    <property name="text">
     <string> select </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Product Reviews:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_review">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Product Review Summarization:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_summary">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_previous">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.985, y1:0.558, x2:1, y2:0, stop:0.189055 rgba(255, 255, 255, 255), stop:0.721393 rgba(200, 226, 246, 255));</string>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>670</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.985, y1:0.558, x2:1, y2:0, stop:0.189055 rgba(255, 255, 255, 255), stop:0.721393 rgba(200, 226, 246, 255));</string>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_review_number">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_open">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_select">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_summary">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>670</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.985, y1:0.558, x2:1, y2:0, stop:0.189055 rgba(255, 255, 255, 255), stop:0.721393 rgba(200, 226, 246, 255));</string>
    </property>
    <property name="text">
     <string>show summary</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
